Makefile

```diff
@@ -25,20 +25,20 @@ get_wax_blockchain: make_deps_dir
         git fetch --all --tags && \
         git checkout $(WAX_BRANCH); \
     fi && \
-    git checkout tags/$(WAX_VERSION) && git submodule update --init --recursive
```

Comment:
@danielvo11 I think we want to checkout the specific tag here no?

Reply:
We already checkout branch in line 21 and 26. WAX_VERSION is version of docker image, not version of code.

Reply:
Right, I think the WAX_BRANCH and WAX_VERSION fields are actually competing to do the same thing.I think we should be setting **WAX_BRANCH=v4.0.1wax01**, and this will actually be correct